docs/reference/scripting/security.asciidoc

```diff
@@ -36,7 +36,7 @@ configured to run both types of scripts. To limit what type of scripts are run,
 set `script.allowed_types` to `inline` or `stored`. To prevent any scripts from 
 running, set `script.allowed_types` to `none`.
 
-IMPORTANT: If you use {kib}, set `script.allowed_types` to `both` or `inline`.
```

Comment:
`stored` instead of `both`.

Reply:
I think the point is that you should set it to `inline` or `"inline,stored"` (both), because Kibana requires at least `inline`.

Reply:
Aye, indeed.